htmlToElement.js

```diff
@@ -22,28 +22,38 @@ function htmlToElement(rawHtml, opts, done) {
         if (rendered || rendered === null) return rendered
       }
 
-      if (node.type == 'text') {
+      if (node.type === 'text') {
+        var data = node.data;
+        var textTransformation = opts.styles[parent.name] ? ReactNative.StyleSheet.flatten( opts.styles[parent.name] ).textTransformation : null;
+        switch(textTransformation) {
+          case 'uppercase':
+            data = data.toUpperCase();
+            break;
+          case 'lowercase':
+            data = data.toLowerCase();
+            break;
+        }
         return (
-          <Text key={index} style={parent ? opts.styles[parent.name] : null}>
-            {entities.decodeHTML(node.data)}
+            <Text key={index} style={parent ? opts.styles[parent.name] : null}>
+              {entities.decodeHTML(data)}
           </Text>
         )
       }
 
-      if (node.type == 'tag') {
+      if (node.type === 'tag') {
```

Comment:
I appreciate `===` is in more common use, but since this is string comparison let's stick with the project's existing style. If you feel strongly this should be changed, submit a different PR :)